There're additional build failure.
```
[ERROR] /home/travis/build/prestosql/presto/presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduMetadata.java:[202,34] error: [MissingOverride] getTableMetadata implements method in ConnectorMetadata; expected @Override
    (see https://errorprone.info/bugpattern/MissingOverride)
  Did you mean '@Override public ConnectorTableMetadata getTableMetadata(ConnectorSession session, ConnectorTableHandle tableHandle)'?
```
https://travis-ci.com/prestosql/presto/builds/135581475

--- a/presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduInsertTableHandle.java
+++ b/presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduInsertTableHandle.java
@@ -74,4 +74,12 @@ public class KuduInsertTableHandle
     {
         return false;
     }
+
+    public KuduTable getTable(KuduClientSession session)
+    {
+        if (table == null) {
+            table = session.openTable(schemaTableName);
+        }
+        return table;
+    }
 }

--- a/presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduMetadata.java
+++ b/presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduMetadata.java
@@ -199,6 +199,7 @@ public class KuduMetadata
         }
     }
 
+    @Override
     public ConnectorTableMetadata getTableMetadata(ConnectorSession session, ConnectorTableHandle tableHandle)
     {
         KuduTableHandle kuduTableHandle = (KuduTableHandle) tableHandle;

